SPECS/sm.spec

```diff
@@ -40,7 +40,16 @@ Obsoletes: sm-additional-drivers
 
 # XCP-ng patches
 # Generated from our sm repository
-# git format-patch v2.46.16..2.46.16-8.3
+# git format-patch v3.0.3-xcpng..3.0.3-8.3
+# Note: the v3.0.3-xcpng tag was manually created by us on our fork because
+# the upstream sm doesn't provide maintenance updates anymore
+# To create this tag in the sources, you must create a 3.0.3-8.3 branch from the
+# previous -xcpng tag then cherry pick each upstream commit referenced in the changelog
+# of the upstream spec file.
+# To ensure you have all changes, you can use:
+# `diff -urq <sources> <upstream sources>`.
+# After that we can create the tag: `git tag -a v3.0.3-xcpng -m "v3.0.3-xcpng"`,
+# push the commits and tag.
```

Comment:
Did you really have to cherry-pick all commits? I think they pushed all commits but forgot to push the tags from their true main repo.